core/templates/pages/contributor-dashboard-admin-page/services/contributor-dashboard-admin-stats-backend-api.service.ts

```diff
@@ -47,6 +47,7 @@ export interface TranslationSubmitterBackendDict {
   rejected_translations_count: number;
   rejected_translation_word_count: number;
   first_contribution_date: string;
+  first_contributed_in_days: number;
```

Comment:
Where do you use this field?

Reply:
removed it. actually it was used in the previous implementation.